internal/interfaces/grpc/service.go

```diff
@@ -220,11 +230,22 @@ func NewService(opts ServiceOpts) (interfaces.Service, error) {
 		}
 	}
 
+	var walletPassword string
+	if opts.WalletUnlockPasswordFile != "" {
+		walletPasswordBytes, err := ioutil.ReadFile(opts.WalletUnlockPasswordFile)
+		if err != nil {
+			return nil, err
+		}
+
+		walletPassword = strings.TrimSpace(string(walletPasswordBytes))
```

Comment:
the unlockerd service uses:
```
 bytes.TrimFunc(walletPasswordBytes, func(r rune) bool {
	return r == 10 || r == 32
}), nil
```
this function trims both spaces and end-of-lines. I would keep using this trim function.

Reply:
Done